CactbotOverlay/resources/dana/bars/bars.js

```diff
@@ -172,9 +165,12 @@ class ComboTracker {
     this.comboTimer = null;
 
     if (nextState == null) {
-      this.considerNext = [];
+      this.considerNext = this.startList;
     } else {
-      this.considerNext = this.comboNodes[nextState].next;
+      this.considerNext = [];
+      Array.prototype.push.apply(this.considerNext, this.comboNodes[nextState].next);
+      Array.prototype.push.apply(this.considerNext, this.startList);
```

Comment:
It might make sense to clear the combos on job change and have OnJobChange() do AddCombo() for everything for the given job.